content/applications/hr/payroll/payroll_overview.rst

```diff
@@ -0,0 +1,90 @@
+
+.. image:: payroll_overview/contracts-overview.png
+   :align: center
+   :alt: Contracts dashboard view showing running contracts and contracts with issues.
+
+Work Entries
+============
+
+The :guilabel:`Work Entries` dashboard provides a visual overview of all employee's individual time
+sheets, with each day split into morning and afternoon shifts. From this page, work entries can be
+regenerated if there is an update to schedules, and payslips can be generated.
+
+.. image:: payroll_overview/work-entries-overview.png
+   :align: center
+   :alt: Work Entries dashboard view showing all employee's work entries.
+
+Conflicts
+---------
+
+Any work entry that has a conflict to be resolved is indicated on the main :guilabel:`Work Entry`
+dashboard, or by going to the separate :guilabel:`Conflicts` dashboard. A conflict appears for any
+request that has not been approved, such as sick time or vacation, or if there are any errors on the
+work entry, such as required fields being left blank. Conflicts are required to be resolved before
+payslips can be generated.
+
+Payslips
+========
+
+:guilabel:`Payslips` consists of three sections; :guilabel:`To Pay`, :guilabel:`All Pay Slips`, and
+:guilabel:`Batches`. :guilabel:`To Pay` displays the payslips that have not been generated yet, and
+can be created from this dashboard.:guilabel:`All Payslips` displays all payslips regardless of
+status, and are organized by month. Both :guilabel:`To Pay` and :guilabel:`All Payslips` display all
+the detailed information for each payslip. :guilabel:`Batches` displays all the payslip batches that
+have been created, and is where commission payslips are generated.
+
+.. image:: payroll_overview/to-pay.png
+   :align: center
+   :alt: Payslips dashboard view showing payslips needing to be paid.
+
+Reporting
+=========
+
+The :guilabel:`Reporting` dashboard is extremely versatile and can display metrics based on the
+:guilabel:`Measures` selected to be displayed, such as basic wages, work days, unforeseen absences,
+and more. The information can be grouped in a variety of ways  selected in the dashboard. :guilabel:
+`Meal Vouchers` provides an overview of  the meal vouchers used by employees, and can be shown by
+day, week, month, quarter, or year. :guilabel:`Attachment of Salary` is a report that shows any
+deductions or allocations per employee, such as child support payments. Other reports are country
+specific, and if the specific report does not apply to the business, a prompt will alert that the
+report is not valid. All reports can be inserted into a spreadsheet for further use.
+
+.. image:: payroll_overview/reporting.png
+   :align: center
+   :alt: Report dashboard view.
+
+Settings
+========
+
+The :guilabel:`Settings` section is where localization settings are configured, including a detailed
+view of all  benefits provided to employees. Whether or not payslips should be posted in accounting,
+or if SEPA payments are created, is configured here.
```

Comment:
Thanks again for the review. Again, I was not sure how detailed to go, since for each section I want a "see also "this doc""  which would go in depth with every single section- but since they don't exist I can't link it to them. I tried to add that as a placeholder in the employees doc but of course I get a github error since the doc isn't published, so I can't link it. I tried to go halfway- a but ore detailed, but not 100% overlapping what I think should be in a separate doc. Hopefully that helps!

Reply:
Same comment @larm-odoo : https://github.com/odoo/documentation/pull/4003#discussion_r1162063199